### PR TITLE
Preparing flexible header, adding Poppins font

### PR DESCRIPTION
**Pull request by gp231982**
A flexible header for all devices was created, all comments have been removed, corrections of styling, "Poppins" font was added to HTML.

### CODE REVIEW

File: public/index.html
Comment (KamilJustynski) [inline public/index.html:1]: The font is added to index and GlobalStyles so you can remove it from anywhere.
We even got it through the conflict.

File: src/common/Header/styled.js
Comment (KamilJustynski) [inline src/common/Header/styled.js:1]: Convert all color names from hexadecimal to theme names so there is consistency. Those that aren't there, fill them in, same with media queries. :)

**Review by KamilJustynski — CHANGES_REQUESTED** [review]

File: public/index.html
Comment (gp231982) [reply public/index.html:1]: I've tried to add it to GlobalStyles but from some reason It did not work for me... that is why I've added it to GlobalStyle. I'll try again.

File: src/common/Header/styled.js
Comment (gp231982) [reply src/common/Header/styled.js:1]: Ok, clear. Let You know when finished.

Comment (gp231982) [reply src/common/Header/styled.js:1]: Changes will be added later acc. to agreements

**Review by KamilJustynski — APPROVED** [review]

File: src/common/Header/styled.js
Comment (KamilJustynski) [reply src/common/Header/styled.js:1]: ok

**Pull request merged**